docker/ci/Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM golang:1.7.3-alpine
+FROM golang:1.7.4-alpine
 
 RUN apk --no-cache add bash clang git g++ make nodejs openjdk8 python perl \
-    ruby ruby-bigdecimal ruby-bundler ruby-io-console ruby-irb ruby-json \
+    ruby ruby-bigdecimal ruby-bundler ruby-io-console ruby-irb ruby-json openssl ca-certificates \
+    && echo "http://dl-cdn.alpinelinux.org/alpine/v3.5/main" >> /etc/apk/repositories \
+    && apk --no-cache add "protobuf>3.1.0" \
+    && go get -u github.com/golang/protobuf/proto \
+    && go get -u github.com/golang/protobuf/protoc-gen-go \
+    && go get -u google.golang.org/grpc \
+    && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub \
+    && wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/glibc-2.23-r3.apk \
```

Comment:
Do you mind also adding a todo in here about the issue you mention in the commit message? Something like

```
TODO: remove third-party protobuf package when google's release works with musl (next release)
```